src/Weikio.ApiFramework.Core/Cache/ApiCacheOptions.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Weikio.ApiFramework.Abstractions;
+
+namespace Weikio.ApiFramework.Core.Cache
+{
+    public class ApiCacheOptions
+    {
+        public int ExpirationTimeInSeconds { get; set; } = 60;
```

Comment:
Rename this to `ExpirationTime` and change the type to `TimeSpan`.

Reply:
I think the idea here was that using seconds is easier than TimeSpans if we are configuring things through appsettings.json or some other json-file.

But yep, this should be changed. TimeSpan can be written as 00:00:60 in appsettings.

Reply:
Done.